script/Arcueid/Day 8/08_00_ARC08_6.txt

```diff
@@ -76,12 +76,13 @@ Then I would have contacted her at the very least, but the thought of communicat
 [sha:158120f0784ae304a29f1b463906d902146c93c5]{
 -- Page 294, Offset 20404.
 -- 次からはちゃんと時間を厳守する。今回は大目に見てくれ」
-I'll be sure to not be late next time. So please let me off the hook this time around."
+I'll make sure not to be late next time. So please let me off the hook this time."
 }
 [sha:9d175f29f418f8895835afc08b34845b3de2152f]{
 -- Page 295, Offset 20405.
 -- 「―――もう。これから殺し合いに行くのに、ぜーんぜん自覚がないみたいね、あなたは」
-"―――Geez. We're going to go out and kill after this, but your head doesn't seem to be in this at all."
+// better suggestions for 殺し合い?
+"―――Geez. even though we're going to kill after this, your head doesn't seem to be in this at all."
```

Comment:
"even though we _need_ to kill after this" maybe? I don't have anything fantastic off the cuff for this.

Reply:
Remember to capitalise 'even'.